src/main/kotlin/uk/gov/justice/digital/hmpps/subjectaccessrequestworker/client/DocumentStorageClient.kt

```diff
@@ -161,7 +160,7 @@ class DocumentStorageClient(
     val fileSize: Int? = null,
     val fileHash: String? = null,
     val mimeType: String? = null,
-    val metadata: Map<String, Any>? = null,
+    val metadata: Any? = null,
```

Comment:
This should be a map, but the original implementation was setting to an Int - to maintain backwards compatibility and flexibility going forward have made this `Any?`.